homework29/FootballTeam.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<title>Untitled Document</title>
+<link  href="style.css" rel="stylesheet" type="text/css" />
+
+<style>
+table, td, th {
+    border-style: thin;
+}
+
+table, tr, td {
+   border: 0.5px solid black;
+}
+
+
+
+td {
+    text-align:center;
+}
+	
+</style>
+
+</head>
+
+<body>
+<div id="Main">
+
+  <div id="Logo">
+    	<div id="Logo1"><img src="bayernLogo.png" width="120" height="120" />
+     </div><!--end the Logo1 div-->
+    </div><!--end the Logo div-->
+    
+    <div id="Menu">
+	<ul>
+      	<li><a href="file:///Users/ibrisimovicnarena/Desktop/Fudbalski%20tim/FootballTeam.html" class="Link">Home</a></li>
+         <li><a href="" class="Link">News</a></li>
+         <li><a href="" class="Link">Teams</a></li>
+         <li><a href="" class="Link">Matches</a></li>
+         <li><a href="" class="Link">Club</a></li>
+         <li><a href="file:///Users/ibrisimovicnarena/Desktop/Fudbalski%20tim/History.html" class="Link">History</a></li>
+         <li><a href="" class="Link">Fans</a></li>
+         <li><a href="file:///Users/ibrisimovicnarena/Desktop/Fudbalski%20tim/trophies.html" class="Link">Trophies</a></li>
+        </ul>   
+    </div><!--end the Menu div-->
+    
+    <div id="Slide"><img src="bigPicture.jpg" width="940" height="438" style=" margin-left: 10px; margin-top:10px;"/>
+    </div><!--end the Slide div-->
+    
+    <div id="Slike">
+   	  <div class="SlikaTop">
+	    <div class="slika1" >
+        
+         	<div class="slika1Bijela">
+           	  
+            <div class="photo">
+               <a href="https://www.youtube.com/watch?v=t53gv9whoRU"><img src="bayern_match.jpg" width="210" height="256" /> </a>				
+               </div><!--end the photo div-->
+                
+         	</div><!--end the slika1Bijela div-->
+            
+        </div><!--end the slika1 div-->
+        
+        	<div class="slika1">
+            
+         	<div class="slika1Bijela">
+                <div class="photo">
+                <a href="https://www.youtube.com/watch?v=ddHiZ34UncM"><img src="barcelona_bayern.jpg" width="210" height="256" /></a>
+                 </div><!--end the photo div-->
+
+            </div><!--end the slika1Bijela div-->
```

Comment:
Kada se dobra identacija koristi, nema potrebe za ovim komentarima "end the ... div" formatiraj code na nekom od online HTML formatera.